src/mpid/ch3/include/mpidimpl.h

```diff
@@ -346,6 +346,8 @@ extern MPIDI_Process_t MPIDI_Process;
 #define MPIDI_REQUEST_TYPE_GET_ACCUM_RESP 13             /* target is sending GACC response data */
 #define MPIDI_REQUEST_TYPE_FOP_RECV 14                   /* target is receiving FOP data */
 #define MPIDI_REQUEST_TYPE_FOP_RESP 15                   /* target is sending FOP response data */
+/* Request types for persistent collectives */
+#define MPIDI_REQUEST_TYPE_PERSISTENT_BCAST 16
```

Comment:
Let's move this change to the same commit that expands the number of ch3 bits, more coherent that way.